src/Management/test/Endpoint.Test/ManagementEndpointServedOnDifferentPort.cs

```diff
@@ -0,0 +1,161 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Net;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Steeltoe.Common.Hosting;
+using Xunit;
+
+namespace Steeltoe.Management.Endpoint.Test;
+
+public class ManagementEndpointServedOnDifferentPort
+{
+    public ManagementEndpointServedOnDifferentPort()
+    {
+        ClearEnvVars();
```

Comment:
Does this need to be called at the start and end of every test in this class? We could [use IDisposable](https://xunit.net/docs/shared-context) to clean up instead of calling it directly each time

Reply:
Its mostly with the start - I sometimes added this at the end, but only some tests are sensitive to this and I added it to the either start or in this case constructor.